ui/admin/tests/helpers/selectors.js

```diff
@@ -10,6 +10,7 @@ export const EDIT_BTN = '.rose-form-actions [type=button]';
 export const FIELD_NAME = '[name=name]';
 export const FIELD_NAME_VALUE = 'Random name';
 export const FIELD_NAME_ERROR = '[data-test-error-message-name]';
+export const FIELD_TEXT_ERROR = '.hds-form-error__message';
```

Comment:
I don't think we actually need this selector. The `FIELD_NAME_ERROR` should be the selector for finding the error. If that `data-test` attribute isn't in the template file, you can simply add it.

Reply:
It is supposed to be specifically related to name. Any error we need to act or assert against should have an explicit error selector. I explained this more in the other [PR](https://github.com/hashicorp/boundary-ui/pull/2693#issuecomment-2669582232).

Reply:
+1 to the last comment.